frontend/src/i18n/locales/en.yaml

```diff
@@ -123,6 +123,9 @@ video:
   ended: Ended
   license: License
   source: Source
+  embed:
+    button: einbetten
+    title: Video einbetten
```

Comment:
Ehm... you swapped the German and English translations!

Reply:
Yup. xD